backend/src/index.js

```diff
@@ -125,6 +125,65 @@ app.get(
   })
 )
 
+app.get(
+  '/set-status/:status/:id',
+  errorWrap(async (req, res) => {
+    try {
+      let response = "Status set Sucessfully"
+      switch(req.params.status) {
+        case "pending":
+          await Candidate.findByIdAndUpdate(req.params.id, {status: "pending"});
+          break;
+        case "accepted":
+          await Candidate.findByIdAndUpdate(req.params.id, {status: "accepted"});
+          break;
+        case "interviewing":
+          await Candidate.findByIdAndUpdate(req.params.id, {status: "interviewing"});
+          break;
+        case "rejected":
+          await Candidate.findByIdAndUpdate(req.params.id, {status: "rejected"});
+          break;
+        default:
+          response = "Invalid status, please try again"
+      }
+      res.json({result: response})
+    } catch (err) {
+      res.status(400).json({ message: err.message })
+    }
+  })
+)
+
+// Following function doesn't work, not sure why...
+app.get(
+  '/initialize-status',
+  errorWrap(async (req, res) => {
+    try {
+      Candidate.update({}, { $set: { status: 'pending' }})
+      res.send("Done updating")
+    } catch (err) {
+      res.status(400).json({ message: err.message })
+    }
+  })
+)
+
+app.get(
+  '/filter-status/:status',
```

Comment:
this pulls from REST resource naming. Since you are returning certain candidates based off of their status. The URL endpoint should be part of /candidates, where status is one of the query parameters (stuff that's after `?` ) It would be something like `/candidates?status=pending` 

http://www.restapitutorial.com/lessons/restquicktips.html are some quick tips on dsigning a RESTful api